package/benchmarks/bbob_largescale/README.md

```diff
@@ -0,0 +1,65 @@
+---
+author: Optuna team
+title: The blackbox optimization benchmarking largescale (bbob-largescale) test suite
+description: The blackbox optimization benchmarking largescale (bbob-largescale) test suite consists of high-dimensional 24 noiseless single-objective test functions including Sphere, Ellipsoidal, Rastrigin, Rosenbrock, etc. This package is a wrapper of the COCO (COmparing Continuous Optimizers) experiments library.
+tags: [benchmark, continuous optimization, high-dimensional, BBOB, COCO]
+optuna_versions: [4.1.0]
+license: MIT License
+---
+
+## Abstract
+
+The blackbox optimization benchmarking largescale (bbob-largescale) test suite comprises high-dimensional 24 noiseless single-objective test functions. Each benchmark function is provided in dimensions \[20, 40, 80, 160, 320, 640\] with 15 instances. Please refer to [the paper](https://arxiv.org/abs/1903.06396) for details about each benchmark function.
+
+## APIs
+
+### class `Problem(function_id: int, dimension: int, instance_id: int = 1)`
+
+- `function_id`: [ID of the bbob benchmark function](https://numbbo.github.io/coco/testsuites/bbob-largescale) to use. It must be in the range of `[1, 24]`.
+- `dimension`: Dimension of the benchmark function. It must be in `[20, 40, 80, 160, 320, 640]`.
+- `instance_id`: ID of the instance of the benchmark function. It must be in the range of `[1, 15]`.
```

Comment:
I could not confirm that there exist only 15 instances from the literature, but seems there are only 15 instances if I guess from Section 3.1

- https://inria.hal.science/hal-02067932/file/2019_GECCO_Mixint_Suites.pdf

Reply:
I wrote this section based on [the COCO's implementation](https://github.com/numbbo/coco-experiment/blob/a1bde78151a139e0267fb7c8940c1f401c6c363a/src/suite_largescale.c#L41-L53) that actually provides 15 instances.